<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPointMon</class>
 <widget class="QDialog" name="CPointMon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tablePoint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>1900</width>
     <height>1050</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕_코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="rowCount">
    <number>15</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>순번</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>태그</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>태그설명</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>경보</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>상태</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>현재값</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelPage">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>Page(0/0)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFirst">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>Group 선택안됨</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPre">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>1640</x>
     <y>10</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLast">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="control">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>10</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>제어</string>
   </property>
  </widget>
  <widget class="QPushButton" name="range">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>10</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>속성</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>CPointMon</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>929</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
